test/Github/Tests/Api/RateLimitTest.php

```diff
@@ -37,11 +37,9 @@ class RateLimitTest extends TestCase
     protected $api;
 
     /**
-     * Used to construct common expectations for the API input data in each unit test.
-     *
-     * {@inheritdoc}
+     * @before
      */
-    protected function setUp(): void
+    public function initMocks()
```

Comment:
Just curious why this change and not just keep the setUp?

Reply:
So we don't get messed about by phpunit's own type signatures. It's always a pain when upgrading.

Reply:
Good 👍